### PR TITLE
feat(e2e/knuu): use latest release and prepare for node knuu tests

### PR DESCRIPTION
**Pull request by smuu**
Closes https://github.com/celestiaorg/celestia-app/issues/4075

## Overview

This PR:
- Uses the latest release of knuu with fixes
- Prepares the testnet to be used by celestia-node

### CODE REVIEW

**Review by rootulp — DISMISSED** [review]

**Review by ninabarbakadze — COMMENTED** [review]
thanks sam! lint needs fixing before it can merge

File: test/e2e/testnet/testnet.go
Comment (evan-forbes) [reply test/e2e/testnet/testnet.go:411]: I don't want to block on this @smuu if we're close on time, but these logic statements make maintaining this code more cumbersome than is required

Comment (smuu) [reply test/e2e/testnet/testnet.go]: fixed

File: test/e2e/testnet/testnet.go
Comment (smuu) [reply test/e2e/testnet/testnet.go]: fixed

File: test/e2e/main.go
Comment (smuu) [reply test/e2e/main.go]: fixed

**Review by evan-forbes — APPROVED** [review]
:+1:

**Review by rootulp — APPROVED** [review]

**Pull request merged**
